ansible/roles/geoblacklight/tasks/geoblacklight_setup.yml

```diff
@@ -6,7 +6,6 @@
     owner: "{{ project_user }}"
     group: "{{ project_group }}"
     state: directory
-    recurse: yes
```

Comment:
I removed the `recurse: yes` because this applied the permissions recursively, too.  You are correct in your comment about `recurse: yes` also changing ownerships when we don't really want this. This change also goes hand in hand with the `become_user: "{{ project_user }}"` on the Git clone operation, which is also new.  That causes the directories to be owned by `{{ project_user }}`, something the `recurse: yes` was originally included to achieve.

This way, we can be more flexible about setting the directory permissions in the code repo (which Git does maintain, IIRC), which may be a bit easier that adding a lot of `file:` blocks to be very explicit about their settings.

In this set of changes, I tend to reserve the `file:` blocks for those cases when the ownership and permissions should be assigned to the `{{project_run_user}}`.